spec/takeaway_spec.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/takeaway'
+
+describe Takeaway do
+  let (:test_menu) { described_class.new }
+
+  describe 'takeaway_menu' do
+    it 'prints a default list of dishes with prices' do
```

Comment:
Good approach to test the menu dishes. It sounds like this is the one you couldn't get working according to your readme. Could you use a "before do" block or maybe set up another let/allow at the top of the test?